content/pt-br/docs/concepts/workloads/controllers/ttlafterfinished.md

```diff
@@ -0,0 +1,57 @@
+---
+reviewers:
+  - janetkuo
+title: Automatic Clean-up for Finished Jobs
+content_type: concept
+weight: 70
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
+
+TTL-after-finished {{<glossary_tooltip text="controller" term_id="controller">}} provides a
+TTL (time to live) mechanism to limit the lifetime of resource objects that
+have finished execution. TTL controller only handles
+{{< glossary_tooltip text="Jobs" term_id="job" >}}.
+
+<!-- body -->
+
+## Controlador TTL-after-finished
+
+O controlador TTL-after-finished é suportado apenas para Trabalhos. Um operador de cluster pode usar esse recurso para limpar
```

Comment:
are we using the literal translation of `jobs` to `Trabalhos`? should we keep `job` and not translate that at all?

Reply:
We have standardized the translation of "job" as "tarefa" in Brazilian Portuguese.

Reply:
perfect sounds great